api/app/signals/apps/signals/models/routing_expression.py

```diff
@@ -14,6 +17,7 @@ class RoutingExpression(models.Model):
         related_name='routing_department'
     )
     _department = models.ForeignKey(Department, on_delete=models.CASCADE)
+    employee = models.ForeignKey(to=User, on_delete=models.SET_NULL, blank=True, null=True)
```

Comment:
A few remarks:

* Also why is it called `employee`? On a Signal it is called `user_assignment`. Therefore it should be called something like `_user`.
* Now it is possible to assign a department and a user that is not part of the department. This can result in the signal not being visible/editable for the assigned user. This because acces to the signal is granted based on the department of a user. So there could be a mismatch here. This should not be possible.
* Should it also be possible to only assign a user and not a department?
* This should be an Expression on its own instead of being part of the `routing_departments`

Reply:
I will rename it to _user.
For point 2 and 3 i'll refer to the issue: https://github.com/Signalen/product-steering/issues/70
Here is stated:

> Tevens is afgesproken dat de lijst met medewerkers niet afhankelijk wordt gemaakt van de eerder gekozen afdeling, er vind tussendoor dus géén verversing plaats. Het is aan de beheerder om een afdeling-medewerker combinatie te selecteren die ook daadwerkelijk bestaat. Het testen zal moeten uitwijzen wat er gebeurd wanneer een afdeling-medewerker combinatie gekozen wordt die niet kan (worden waarden dan toch gevuld, wordt routeerregel niet uitgevoerd of wordt bijv. alleen afdeling ingevuld).

And ill make sure to move it from routing_departments

Reply:
@vanbuiten after changing this i figure i need to run the migrations command again.
Seeing there are multiple comments about migrations what is the best way to handle this?